### DISCUSSION TITLE
Error when seeding database

### DISCUSSION THREAD

**Discussion by mamob07** [firefly-iii/firefly-iii#9508] [Firefly III] [answered]
What is this error? I get this when seeding the database via `php artisan migrate --seed` after upgrading to Firefly III v6.1.24 from 6.1.21.

![{D942314B-F024-4D86-9490-2B37BDB4781C}](https://github.com/user-attachments/assets/807e4bab-a4e2-4676-a7e9-489a72812cc7)

**Comment by JC5** [comment, marked as answer]
You seem to be missing the mbstring extension.
